scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1340,6 +1481,22 @@ def postingGateJson(pipelineJson, cookie):
         raise e
 
 
+def get_redis_conn():
+    if spin_db_type == 'redis':   
+        #Establishing the redis connection       
+        redis = redis.Redis(host=redis_host, port=redis_port, password=redis_password)
+        print("Redis connection established successfully")
+        return redis
+
+
+def get_sql_db_conn():
+    if spin_db_type == 'sql':
+        #Establishing the spinnaker sql database connection       
+        sqldb = mysql.connector.connect(database='orca', user=spin_db_username, password=spin_db_password, host=spin_db_host)
```

Comment:
@sudhakaropsmx  for other daatbases, we are accepting db name as an input parameter but here it is hard coded. On this Gopi once mentioned there might be situation we have a db different name being used in cusotmer env thought that is unusual but possible. Do you think same applies here as well for spinnaker ? the db name might be different than the default orca ?

Reply:
Default names will used in spinnaker database